src/main/java/ch/bfh/bti7081/s2018/green/models/entities/Program.java

```diff
@@ -0,0 +1,77 @@
+package ch.bfh.bti7081.s2018.green.models.entities;
+
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Period;
+
+public class Program {
+
+    private Therapy therapy;
+
+    private LocalDate start;
+
+    private LocalDate stop;
+
+    private Period frequence;
+
+    private RecurringEvent firstEvent;
+
```

Comment:
@cyrillbolliger : We might think about removing the constructor's last parameter "RecurringEvent" and just rely on the method setNext(RecurringEvent event). Actually that's what Raphael already does in his method createRecuringEvents(RecurringEvent event) when he instantiates new RecurringEvents setting the constructor's last parameter == null. I suggest that we discuss that on Monday (face-to-face).